src/lib/validation/legacy-input-normalizers.js

```diff
@@ -22,9 +24,9 @@ export const addToCartNormalizer = (input : any) => {
     } else if (input.cartTotal) {
       input.total = input.cartTotal;
       delete input.cartTotal;
+      input = limitCartItems(input);
```

Comment:
Just pointing out that if the user uses 'total' _and_ has more than 10 items they will never be truncated.

Talked with Song. I think it'd be ok to include this check in both cases.

Reply:
ah okay, I'll also add it in the other case for if they have a total value then

Reply:
or put this after the if statement?
```
if (input) {
  ....
  input = limitCartItems(input)
}
```